src/cli/task-build.ts

```diff
@@ -24,10 +24,7 @@ export const taskBuild = async (coreCompiler: CoreCompiler, config: d.ValidatedC
     const compiler = await coreCompiler.createCompiler(config);
     const results = await compiler.build();
 
-    // TODO(STENCIL-148) make this parameter no longer optional, remove the surrounding if statement
-    if (sys) {
-      await telemetryBuildFinishedAction(sys, config, coreCompiler, results);
-    }
+    await telemetryBuildFinishedAction(config.sys, config, coreCompiler, results);
```

Comment:
There's an argument to be had here that since `sys` exists on `config`, we could update the function signature of `telemetryBuildFinishedAction` so that calling it looks something like:
```diff
- await telemetryBuildFinishedAction(config.sys, config, coreCompiler, results);
+ await telemetryBuildFinishedAction(config, coreCompiler, results);
```

I decided _not_ to go down this route for 2 reasons. 
1.  I don't think we have a good feel for what we want to do from a consistency standpoint (if anything) throughout the codebase
2. This would have grown in scope into updating additional functions in `telemetryBuildFinishedAction`, which I felt was out of scope here

Reply:
makes sense — I've made changes along those lines in a few places and I think I generally think they're a good idea, but we don't need to always do them and I think making changesets more focused and small is basically always a good idea 👍